best_practices/business-logic.rst

```diff
@@ -151,8 +152,6 @@ the class namespace as a parameter:
         app.slugger:
             class: '%slugger.class%'
 
-This practice is cumbersome and completely unnecessary for your own services:
```

Comment:
We can instead reword this: "Using a parameter for the class name of your own services is cumbersome and completely unnecessary"